common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -194,7 +194,7 @@ class MemoryCache(object):
     the modulestore, and stores the data in a dictionary in memory.
     """
     def __init__(self):
-        self._data = {}
+        self.data = {}
```

Comment:
**Note to reviewers:** Renaming from `_data` to `data` allows this stub cache object to be used as a request_cache in the modulestore.